source/deploy/desktop-app-changelog.rst

```diff
@@ -55,6 +55,7 @@ Open Source Components
 Known Issues
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+- Desktop App Windows 10: The taskbar does not flash on receipt of a new message when the setting is enabled `MM-58087 <https://mattermost.atlassian.net/browse/MM-58087>`_.
```

Comment:
Could we phrase this to say that it may not happen? It only seems to reproduce for certain machines.